src/components/OsmContribution.jsx

```diff
@@ -0,0 +1,30 @@
+/* global _ */
+import React from 'react';
+
+const OsmContribution = ({ poi }) => {
+  const osmContributionBlock = poi.isFromOSM && poi.isFromOSM();
+  if (!osmContributionBlock) {
+    return null;
+  }
+
+  return <div className="osm_contribute">
+    <div className="osm_contribute__logo"></div>
```

Comment:
JSX is like XML, all empty components can use the self closing syntax `<div className="osm_contribute__logo" />`. We can install an ESlint rule for that later.